turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -275,7 +275,7 @@ impl StandardBroadcastRun {
                 is_last_in_slot,
                 process_stats,
                 blockstore::MAX_DATA_SHREDS_PER_SLOT as u32,
-                shred_code::MAX_CODE_SHREDS_PER_SLOT as u32,
+                blockstore::MAX_CODE_SHREDS_PER_SLOT as u32,
```

Comment:
Seeing `blockstore::` qualified out, I now realize / remember the constant is defined in `blockstore.rs`. So while we're looking at this, I think it makes more sense for the constant to live with shreds, maybe in `shred.rs`.

Conceptually, it should be Shred dictating how many shred per block are allowed and Blockstore working with that; not the other way around. Moving the constant helps reinforce that

Reply:
Turbine won't broadcast shreds if index is too big:
https://github.com/anza-xyz/agave/blob/173dc226882f12daaa3975919a89aa98578ac7c5/turbine/src/broadcast_stage/standard_broadcast_run.rs#L277-L278

https://github.com/anza-xyz/agave/blob/173dc226882f12daaa3975919a89aa98578ac7c5/turbine/src/broadcast_stage/standard_broadcast_run.rs#L147-L152

Likewise, shred already has logic to discard shreds based on index:
https://github.com/anza-xyz/agave/blob/173dc226882f12daaa3975919a89aa98578ac7c5/ledger/src/shred.rs#L825-L829

So yes, technically we could lower or raise this limit and shred creation & turbine propagation should work as-is. But, we do have a limit and that limit must be adhered to. And, I still think shred is the best place for that limit to be defined.